contracts/SnapshotToken/Helpers/MTokenAllowanceController.sol

```diff
@@ -23,4 +23,20 @@ contract MTokenAllowanceController {
         internal
         returns (bool allow);
 
```

Comment:
A question not really related to this PR, but why are some interfaces prefixed with M instead of I?

Reply:
because this is internal interface which has internal functions, not public. and this is convention here

solidity has no `virtual` and `override` markers so it does not know if given method can be overriden and that you intend to override...